templates/web/merton/tracking_code.html

```diff
@@ -0,0 +1,7 @@
+<!-- Google Tag Manager -->
```

Comment:
We cannot add this without some form of cookie control/manager to accept/reject these.

Reply:
Apparently this will be handled by the tag manager code, or so we've been told? https://github.com/mysociety/societyworks/issues/2628#issuecomment-934571539

Reply:
Oh, fair enough! As long as that works :)